Zoe + Smart ED - Move static locals to member variables

frogonwheels opened the pull request.

dexterbg commented:
Michael,

I get a build issue with this PR:

```
CXX build/vehicle_smarted/src/ed_can_poll.o
/home/balzer/esp/Open-Vehicle-Monitoring-System-3/vehicle/OVMS.V3/components/vehicle_smarted/src/ed_can_poll.cpp: In member function 'virtual void OvmsVehicleSmartED::IncomingPollReply(canbus*, uint16_t, uint16_t, uint8_t*, uint8_t, uint16_t)':
/home/balzer/esp/Open-Vehicle-Monitoring-System-3/vehicle/OVMS.V3/components/vehicle_smarted/src/ed_can_poll.cpp:264:9: error: 'mlframe' was not declared in this scope
     if (mlframe > 0)
         ^
```

dexterbg commented:
I don't find this change in your master, and I don't find the mlframe variable anywhere, so I'll revert this merge for now.

frogonwheels commented:
Argh. Sorry about that. I’ve fixed the commit.  Yeah – mlframe was going to be a parameter in the future.  This was a lazy commit and I’d forgotten about it.

 

Fixed in  336321406bb69289d6871db491de19f05df0c63b  on that branch in frogonwheels branch on github now.

 

//.ichael

 

From: Michael Balzer ***@***.***> 
Sent: Saturday, August 19, 2023 4:06 PM
To: openvehicles/Open-Vehicle-Monitoring-System-3 ***@***.***>
Cc: Michael ***@***.***>; Author ***@***.***>
Subject: Re: [openvehicles/Open-Vehicle-Monitoring-System-3] Zoe + Smart ED - Move static locals to member variables (PR #905)

 

I don't find this change in your master, and I don't find the mlframe variable anywhere, so I'll revert this merge for now.

—
Reply to this email directly, view it on GitHub <https://github.com/openvehicles/Open-Vehicle-Monitoring-System-3/pull/905#issuecomment-1684887991> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/AABNANXYDZQCKHX6PLL77JTXWBXUZANCNFSM6AAAAAAYHTAVWE> .
You are receiving this because you authored the thread.Message ID: ***@***.***>
